### PR TITLE
Add docs for retention and automated tagging

### PR DESCRIPTION
**Pull request by nickvergessen**
### Retention

> retention

### Auto taggging

> autotagging

### CODE REVIEW

File: admin_manual/file_workflows/automated_tagging.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
+==========================
+Automated Tagging Of Files
```
Comment (MariusBluem) [inline admin_manual/file_workflows/automated_tagging.rst]: „Of“ vs „of“ 🤔

File: admin_manual/file_workflows/retention.rst [hunk 2]
```diff
@@ -0,0 +1,33 @@
+==================
+Retention Of Files
```
Comment (MariusBluem) [inline admin_manual/file_workflows/retention.rst]: Same here 🙈

File: admin_manual/file_workflows/automated_tagging.rst (hunk 1 above)
Comment (nickvergessen) [inline admin_manual/file_workflows/automated_tagging.rst]: yeah okay, the capitalization in headlines is awkward...

Comment (nickvergessen) [inline admin_manual/file_workflows/automated_tagging.rst]: Done

**Pull request merged**